commands/kubernetes.go

```diff
@@ -399,6 +399,7 @@ This command adds the credentials for the specified cluster to your local kubeco
 	AddBoolFlag(cmdSaveConfig, doctl.ArgSetCurrentContext, "", true, "Boolean indicating whether to set the current kubectl context to that of the new cluster")
 	AddIntFlag(cmdSaveConfig, doctl.ArgKubeConfigExpirySeconds, "", 0,
 		"The length of time the cluster credentials will be valid for in seconds. By default, the credentials are automatically renewed as needed.")
+	AddStringFlag(cmdSaveConfig, doctl.ArgKubernetesAlias, "", "", "The context to save authentication information.")
```

Comment:
How about something like:

```suggestion
	AddStringFlag(cmdSaveConfig, doctl.ArgKubernetesAlias, "", "", "An alias for the cluster context name. Defaults to 'do-<region>-<cluster-name>'.")
```

Reply:
thanks!